components/member-profile/active-task/index.js

```diff
@@ -1,19 +1,62 @@
 import PropTypes from 'prop-types';
-import classNames from 'components/member-profile/active-task/active-task.module.scss';
-import { timeWas } from 'helper-functions/time-was';
+import classNames from './active-task.module.scss';
+import { timeWas } from '../../../helper-functions/time-was';
+import { percentageofDaysRemaining } from '../../../helper-functions/taskProgress';
+
 const ActiveTask = ({ taskDetails }) => {
-  const { title, purpose, startedOn, endsOn } = taskDetails;
-  let completedDate = '';
+  const { title, purpose, startedOn, endsOn, percentCompleted } = taskDetails;
+  let completedDate,
+    percentageOfDaysRemaining = '';
   completedDate = timeWas(startedOn * 1000, false, endsOn * 1000);
+
+  let percentOfTaskLeft = 100 - percentCompleted;
+
+  percentageOfDaysRemaining = percentageofDaysRemaining(startedOn, endsOn, completedDate);
+
+  function estimatedDays() {
+    if (percentageOfDaysRemaining >= percentOfTaskLeft) {
+      return classNames.showTextGreen;
+    } else if (percentageOfDaysRemaining < 50 && percentOfTaskLeft > 75) {
+      return classNames.showTextOrange;
+    } else if (percentageOfDaysRemaining < 25 && percentOfTaskLeft > 35) {
+      return classNames.showTextRed;
+    }
+    return classNames.showTextYellow;
+  }
+  function showProgressIndicatorColour() {
+    const estimatedDaysOfTaskComplete = estimatedDays();
+    if (estimatedDaysOfTaskComplete === classNames.showTextGreen) {
+      return classNames.showProgressGreen;
+    } else if (estimatedDaysOfTaskComplete === classNames.showTextOrange) {
+      return classNames.showProgressOrange;
+    } else if (estimatedDaysOfTaskComplete === classNames.showTextRed) {
+      return classNames.showProgressRed;
+    } else {
+      classNames.showProgressYellow;
+    }
+  }
+
```

Comment:
we can use similar approach here as well!

Reply:
creating a helper  function would be a better approach here